src/timepicker/types.js

```diff
@@ -40,3 +42,10 @@ export type TimePickerStateT = {
   /** Internal value of the selected time as an integer since midnight (0) */
   value: ?OptionT,
 };
+
+export type TimePickerDefaultPropsT = {
+  format: '12',
+  step: 900,
+  creatable: false,
+  adapter: DateIOAdapter<Date>,
+};
```

Comment:
Why do we need default props to be typed? Also is it valid Flow to include values like so? I haven't seen that before.

Reply:
This is the only way the default adapter prop can be typed correctly. The tricky part here is that we have a more specific date io adapter type in default props then we have in prop types, so this was the only way I could find to allow for passing in the generic type, but then also have the specific type work by default.

Reply:
Interesting, thanks for explanation. Haven't seen this technique before.